doc/ref/stdlib.html

```diff
@@ -1452,7 +1452,7 @@ <h4 id="manifestJsonMinified">
     <div class="panel">
       <p>
         <em>
-          Available since version 0.18.0.
+          Available in upcoming release.
```

Comment:
Looks like these were changed directly in HTML during the release instead of the source file. We need to update that.